### ISSUE TITLE
Bad exit condition

### ISSUE THREAD

**Issue by stdevPavelmc** [skycoin/skyflash#48]
## Expected Behaviour
The application must shutdown gracefully when asked politely (using the menu or the close button)

## Actual Behaviour
No matter what the app keeps complaining about a timer in use. This lead to no error on the final release app in linux, in Windows you see the "Did this software installed ok?" and something similar on Mac OS 

## Steps to Reproduce the Problem
Using the dev environment decribed on the readme.mf file do this:

  1. Start the software by doing a "python3 skyflash-gui.py"
  1. After the app start just close the app, you will find something like this on the console

```
QObject::startTimer: Timers can only be used with threads started with QThread
Abortado (`core' generado)
```
## Specifications
This affects all O: Linux, Windows, Osx

**Comment by stdevPavelmc** [comment]
Bug found,

The problem is that python is deleting objects in an unpredictable order when it exits and then you can hit this or another less common but also spotted error: `QPixmap: Must construct a QApplication before a QPaintDevice; `

Working on solving that, so far most of the solutions are for pure python + QT5 not for python + QML like we are using.

**Comment by stdevPavelmc** [comment]
Closed via PR #49
